src/mudlet.h

```diff
@@ -320,6 +319,11 @@ class mudlet : public QMainWindow, public Ui::main_window
     int64_t getPhysicalMemoryTotal();
     const QMap<QByteArray, QString>& getEncodingNamesMap() const { return mEncodingNameMap; }
 
+    bool firstLaunch = false;
+    // Needed to work around a (likely only Windows) issue:
+    QString mBG_ONLY_BUTTON_SSHEET;
+    QString mFG_BG_BUTTON_SSHEET;
```

Comment:
The difference between the two - from just looking at their names - isn't clear. Can we improve the name or give better comments?

Reply:
First is a style sheet with background only, second has foreground information, too. 
That's what I get from looking at the names only. Seems fine to me, no?

Reply:
What's a stylesheet with a background and foreground actually mean?

Reply:
It has a `color: XXX;` entry as well as a `background-color: XXX;` one - and is needed fro those cases where we include text as well as a coloured background on a `QPushButton` so that we can force the text to be a contrasting (White or Black) colour...